tree/ntuple/inc/ROOT/RNTupleDescriptor.hxx

```diff
@@ -657,6 +657,11 @@ private:
    //// All fields above are part of the schema and are cloned when creating a new descriptor from a given one
    //// (see CloneSchema())
 
+   std::uint16_t fVersionEpoch = 0; ///< Set by the descriptor builder when deserialized
+   std::uint16_t fVersionMajor = 0; ///< Set by the descriptor builder when deserialized
+   std::uint16_t fVersionMinor = 0; ///< Set by the descriptor builder when deserialized
+   std::uint16_t fVersionPatch = 0; ///< Set by the descriptor builder when deserialized
```

Comment:
We could also store these as a single `uint32_t` (dedicating 1 byte per field) to have only a single value to compare and juggle around

Reply:
I think I see the advantage of having `fVersionEpoch` in its own member, to check that it's actually been set, and then I don't think it's much more complicated to keep the others separate as well.